### PR TITLE
Adding SupplierPartAuxilaryId and option to change operationAllowed

### PR DESCRIPTION
**Pull request by MarioShatterhand**

### CODE REVIEW

**Comment by MarioShatterhand** [comment]
Second commit is also mine, I was just log in on wrong account

File: src/CXml/Models/Messages/PunchOutOrderMessageHeader.php [hunk 1]
```diff
@@ -20,6 +20,10 @@ class PunchOutOrderMessageHeader
 
     /** @var string */
     private $taxDescription;
+    /**
```
Comment (hscheffknecht) [inline src/CXml/Models/Messages/PunchOutOrderMessageHeader.php]: Please add here an empty line

File: src/CXml/Models/Messages/PunchOutOrderMessageHeader.php [hunk 2]
```diff
@@ -20,6 +20,10 @@ class PunchOutOrderMessageHeader
...
+     * @var string
+     */
+    private $operationAllowed;
```
Comment (hscheffknecht) [inline src/CXml/Models/Messages/PunchOutOrderMessageHeader.php]: Could you please add `'create'` as default value for backward compatibility (for the case `operationAllowed` is not set)?

**Review by hscheffknecht — CHANGES_REQUESTED** [review]

File: src/CXml/Models/Messages/PunchOutOrderMessageHeader.php (hunk 2 above)
Comment (MarioShatterhand) [reply src/CXml/Models/Messages/PunchOutOrderMessageHeader.php]: I've added a default value

File: src/CXml/Models/Messages/PunchOutOrderMessageHeader.php (hunk 1 above)
Comment (MarioShatterhand) [reply src/CXml/Models/Messages/PunchOutOrderMessageHeader.php]: Done

**Comment by hscheffknecht** [comment]
Thanks!

**Pull request merged**